github/issues.go

```diff
@@ -21,8 +21,8 @@ type IssuesService service
 //
 // Note: As far as the GitHub API is concerned, every pull request is an issue,
 // but not every issue is a pull request. Some endpoints, events, and webhooks
-// may also return pull requests via this struct. If PullRequestLinks is nil,
-// this is an issue, and if PullRequestLinks is not nil, this is a pull request.
+// may also return pull requests via this struct. The issue is a pull request
+// if IsPullRequest returns true.
```

Comment:
Did you see this comment @kevinburke?

You said "Done", but the "If PullRequestLinks is nil, this is an issue, and if PullRequestLinks is not nil, this is a pull request." sentence is still missing.

Do you feel strongly about removing it? I think it'd be better not to, but if you have reasons to, let me know.